.github/workflows/release.yml

```
@@ -27,16 +28,20 @@ jobs:
           echo "::set-output name=version::$version"
```

```suggestion
          echo "version=$version" >> $env:GITHUB_OUTPUT
```

::set-output is apparently [deprecated](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/).  It was supposed to be removed last year, but that has been "postponed" given widespread use.  No idea if/when it'll actually happen, but can't hurt to remove now.